src/mango/test/mango.py

```diff
@@ -103,12 +135,15 @@ def save_docs_with_conflicts(self, docs, **kwargs):
         r.raise_for_status()
 
     def save_docs(self, docs, **kwargs):
-        body = json.dumps({"docs": docs})
-        r = self.sess.post(self.path("_bulk_docs"), data=body, params=kwargs)
-        r.raise_for_status()
-        for doc, result in zip(docs, r.json()):
-            doc["_id"] = result["id"]
-            doc["_rev"] = result["rev"]
+        for offset in range(0, len(docs), BULK_BATCH_SIZE):
+            chunk = docs[offset : (offset + BULK_BATCH_SIZE)]
+            r = self.sess.post(
+                self.path("_bulk_docs"), data=json.dumps({"docs": chunk}), params=kwargs
+            )
```

Comment:
Tiny style nit: to avoid separate lines for `(` and `)` could just assign `data=json.dumps({"docs": chunk})` and then use `sess.post(self.path("_bulk_docs"), data=data, params=kwargs)`. But if you prefer this style that's fine too.

Reply:
I believe you can also avoid using the `json` library by doing something like e.g.
```python
r = self.sess.post(self.path("_bulk_docs"), json={"docs": chunk}, params=kwargs)
```